modules/delete.js

```diff
@@ -0,0 +1,9 @@
+import displayBook from './displayBook.js';// eslint-disable-line
+import { setMyLib } from './localStorage.js';// eslint-disable-line
+import myLib from '../index.js';// eslint-disable-line
+
+export default function deleteFunc(index) {
+  myLib.splice(index, 1);
+  setMyLib(myLib);
+  displayBook(myLib);
+}
```

Comment:
- [ ] Please, it's one of the project requirements you use `arrow` syntax for functions instead of JavaScript Function Expressions, Kindly make sure to address the requested changes :wink: 💯 

- For more documentation, you can visit this [link](https://www.w3docs.com/learn-javascript/function-expressions.html)